### PR TITLE
feat(pack): add new-procedure-database-oracle-v3

### PR DESCRIPTION
**Pull request by jeanrenemoyap**
## Description

The current monitoring procedure for Oracle Database does not correctly describe the export of the two variables: ORACLE_HOME and LD_LIBRARY_PATH. This request fixes the paths and rewrites all in the new documentation format.

## Target serie

- [x] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

**Comment by garnier-quentin** [comment]
Please update the procedure with last instantclient and oracle dbd versions.

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 1]
```diff
@@ -2,30 +2,88 @@
...
+
+| Metric name      | Description                                                       | Unit |
+| :--------------- | :---------------------------------------------------------------- | :--- |
+| session_used     | The percentage of Oracle session used                             |   %  |
+
+<!--Rman-Backup-Problems-->
+
+| Metric name			   | Description                                                         | Unit   |
+| :----------------------- | :------------------------------------------------------------------ | :----  |
+| rman_backup_problems     | RMAN backup errors number of the server during the last three days  | Count  |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:48]:
```suggestion
| Metric name          | Description                                                         | Unit   |
| :----------------------- | :------------------------------------------------------------------ | :----  |
| #backup_backup_problems     | Number of problems per backup (last 3 days by default)  | Count  |
```

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 2]
```diff
@@ -2,30 +2,88 @@
...
-## RPM
+| Metric name      | Description                                                       | Unit |
+| :--------------- | :---------------------------------------------------------------- | :--- |
+| process_used     | The percentage of Oracle process used                             |   %  |
 
+<!--Datacache-Hitratio-->
+
+| Metric name | Description                                          | Unit |
+| :---------- | :--------------------------------------------------- | :--- |
+| usage       | Check the 'Data Buffer Cache Hit Ratio' of the server|  %    |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:60]:
```suggestion
| Metric name | Description                                          | Unit |
| :---------- | :--------------------------------------------------- | :--- |
| sga_data_buffer_hit_ratio       | Check the 'Data Buffer Cache Hit Ratio' of the server|   %    |
```

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 3]
```diff
@@ -117,45 +176,117 @@ This user account must have the read permission on following tables:
...
+Tablespace 'sysaux' Total: 29.48 GB Used: 527.60 MB (1.90%) Free: 27.88 GB (98.20%)
+Tablespace 'system' Total: 29.48 GB Used: 902.76 MB (3.09%) Free: 27.71 GB (96.91%)
+Tablespace 'temp' Total: 29.48 GB Used: 0.00 B (0.00%) Free: 28.59 GB (100.00%)
+Tablespace 'users' Total: 29.48 GB Used: 2.78 MB (0.01%) Free: 28.48 GB (99.99%)
+```
+
+The above command Check the used space in tablespaces (``` --mode='tablespace-usage' ```) 
+on a specific oracle database installed in the host 10.30.2.38 (``` --hostname='10.30.2.38' ```) 
+It uses Oracle informations (``` --username='SYSTEM' --password='Centreon75' --port='1521' --sid='XE' ```) to connect to the database.
+The check provides a warning if the percentage of used space exceeds 90% and a critical if this percentage exceeds 98%.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:259]: add the option just like you did for Oracle informations :)

**Review by pkriko — CHANGES_REQUESTED** [review]
Hello @jeanrenemoyap see my request changes for the EN part, please report them on the french part.

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md (hunk 3 above)
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md]: Add the options just like you did previously

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 4]
```diff
@@ -117,45 +176,117 @@ This user account must have the read permission on following tables:
...
-| :------------------- | :---------------------------------------------------------------- |
-| ping                 | Monitor host response time                                        |
-| Connection-Number    | Check connection number to the Oracle server                      |
-| Connection-Time      | Check the time to connect to the server                           |
-| Corrupted-Blocks     | Check the number of corrupted blocks in database                  |
-| Datacache-Hitratio   | Check the 'Data Buffer Cache Hit Ratio' of the server.            |
-| Tablespace-Usage     | Check the used space in tablespaces                               |
-| Rman-Backup-Problems | Check RMAN backup errors of the server during the last three days |
-| Tnsping              | Check the connection to a remote listener                         |
+### ```CRITICAL: Cannot connect: (no error string) |```
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md]:
```suggestion
#### ```CRITICAL: Cannot connect: (no error string) |```
```

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 5]
```diff
@@ -117,45 +176,117 @@ This user account must have the read permission on following tables:
...
 
-### Host macro configuration
+This error message means that the Centreon Plugin couldn't successfully connect to the oracle database.
+Check that an Oracle database is installed on this host.
 
-The following macros must be configured on host:
+### ```DBD::Oracle is not root directory |```
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:289]:
```suggestion
#### ```DBD::Oracle is not root directory |```
```

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 6]
```diff
@@ -117,45 +176,117 @@ This user account must have the read permission on following tables:
...
-| Corrupted-Blocks     | Check the number of corrupted blocks in database                  |
-| Datacache-Hitratio   | Check the 'Data Buffer Cache Hit Ratio' of the server.            |
-| Tablespace-Usage     | Check the used space in tablespaces                               |
-| Rman-Backup-Problems | Check RMAN backup errors of the server during the last three days |
-| Tnsping              | Check the connection to a remote listener                         |
+### ```CRITICAL: Cannot connect: (no error string) |```
 
-### Host macro configuration
+This error message means that the Centreon Plugin couldn't successfully connect to the oracle database.
+Check that an Oracle database is installed on this host.
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:287]:
```suggestion
Check that an Oracle database is installed on this host.
Check that no third party device (such as a firewall) is blocking the connection.
```

**Review by pkriko — CHANGES_REQUESTED** [review]

**Review by pkriko — APPROVED** [review]

File: en/integrations/plugin-packs/procedures/applications-databases-oracle.md [hunk 7]
```diff
@@ -85,11 +146,11 @@ Library:
 
 ```bash
 cat > /etc/ld.so.conf.d/oracle.conf <<EOF
-/usr/lib/oracle/12.1/client64/lib/
+/usr/lib/oracle/21.1/client64/lib/
 EOF
 ```
 
-You just have to enter in the file : /usr/lib/oracle/12.1/client64/lib/
+You just have to enter in the file : /usr/lib/oracle/21.1/client64/lib/
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-databases-oracle.md:153]: @pkriko @jeanrenemoyap I think it's not clear. Could you rephrase it please?

**Review by Sims24 — CHANGES_REQUESTED** [review]

**Review by Sims24 — APPROVED** [review]
